fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -110,8 +110,14 @@ private NoarkMetadataFraAvleverendeSakssystem metaDataFrom(MeldingType meldingTy
         metadata.withJournalpostnummer(Integer.valueOf(meldingType.getJournpost().getJpJpostnr()));
         metadata.withJournalposttype(meldingType.getJournpost().getJpNdoktype());
         metadata.withJournalstatus(meldingType.getJournpost().getJpStatus());
-        metadata.withJournaldato(journalDatoFrom(meldingType.getJournpost().getJpJdato()));
-        metadata.withDokumentetsDato(journalDatoFrom(meldingType.getJournpost().getJpDokdato()));
+        String jpJdato = meldingType.getJournpost().getJpJdato();
+        if (!Strings.isNullOrEmpty(jpJdato)) {
+            metadata.withJournaldato(journalDatoFrom(jpJdato));
```

Comment:
Jeg vill brukt emptyToNull her, så man slipper de to IFene.

Reply:
Takkar! Har refaktorert.